### PR TITLE
fix(mangrove.js): remove config dependency

### PR DESCRIPTION
**Pull request by akettal**
This PR removes the `config` dependency which prevent using mangrove.js in a browser context. It follows the temporary removing of the logging feature after the discovery of various issues in the implementation.

It also remove the dependency to `fast-safe-stringify` which also had issue in a browser.

### CODE REVIEW

**Comment by johanleroch** [comment]
Working fine for me now, the dApp isn't broken anymore :)

File: packages/mangrove.js/src/util/config.ts [hunk 1]
```diff
@@ -1,9 +1,10 @@
-import config from "config";
-import path from "path";
-
-const packageConfigDir = path.join(__dirname, "../../../config");
-const baseConfig = config.util.loadFileConfigs(packageConfigDir);
-config.util.setModuleDefaults("MangroveJs", baseConfig);
-
-export default config;
-export { config };
+// FIXME : The config module is not compatible with browser
```
Comment (dontrolle) [inline packages/mangrove.js/src/util/config.ts:1]: I'm generally not a fan of leaving out-commented code in the codebase, unless in a few cases as a temporary measure with a plan to remove it. The code has a tendency to grow stale and non-functional, and can just confuse the reader of the code. The code is still in version control, so we can just pull out the code from there, if we really need it.

With out new issue-scheme, I think we should have a good reason to keep the out-commented code in the codebase, and have a GH issue, which aims to remove it/write code to replace it. Else, let's just remove it.

**Review by dontrolle — COMMENTED** [review]
Overall: Looking good and, as discussed with Johan, alleviates the issues he had earlier. See specific comments on outcommented code.

File: packages/mangrove.js/src/util/config.ts (hunk 1 above)
Comment (akettal) [reply packages/mangrove.js/src/util/config.ts:1]: Right, will remove it.

About commenting code, see the discussion [in this closed PR](https://github.com/mangrovedao/mangrove/pull/227#pullrequestreview-898127552) and the proposal in #228.

Tracking PR is here: https://github.com/mangrovedao/mangrove/issues/236.

File: packages/mangrove.js/src/util/logger.ts [hunk 2]
```diff
@@ -14,8 +14,8 @@
 //   logdataLimiter,
 // } from "@mangrovedao/commonlib-js";
 // import os from "os";
-import safeStringify from "fast-safe-stringify";
-import config from "./config";
+import inspect from "object-inspect";
+// import config from "./config";
```
[the remark this answers is not in the document]
Comment (akettal) [reply packages/mangrove.js/src/util/logger.ts:18]: See https://github.com/mangrovedao/mangrove/pull/232#discussion_r821175746

File: packages/mangrove.js/src/util/logger.ts [hunk 3]
```diff
@@ -36,16 +36,18 @@ import config from "./config";
 
 const stringifyData = (data) => {
   if (typeof data == "string") return data;
-  else return safeStringify(data);
+  else return inspect(data);
 };
 
-const defaultLogLevel = "error";
+// const defaultLogLevel = "error";
```
[the remark this answers is not in the document]
Comment (akettal) [reply packages/mangrove.js/src/util/logger.ts:42]: See https://github.com/mangrovedao/mangrove/pull/232#discussion_r821175746

**Review by dontrolle — APPROVED** [review]
👍

**Pull request merged**